src/mechanics/elasticity.hh

```diff
@@ -188,14 +223,17 @@ public:
 	void update_output_fields();
     
     void set_potential_load(const Field<3, FieldValue<3>::Scalar> &potential)
-    { data_.potential_load = potential; }
+    { eq_fields_->potential_load = potential; }
 
     void calculate_cumulative_balance();
 
 	const Vec &get_solution()
-	{ return ls->get_solution(); }
+	{ return eq_data_->ls->get_solution(); }
+
+	inline EqFields &eq_fields() { return *eq_fields_; }
+
+	inline EqData &eq_data() { return *eq_data_; }
 
-    inline EqData &data() { return data_; }
```

Comment:
Where this was used? Do not see declaration of data_. How it is related to the coupling?